src/styles/blocks/card.scss

```diff
@@ -0,0 +1,76 @@
+body {
+  margin: 0;
+  background-color: gray;
+}
```

Comment:
Styles for the `body` should not be in a file responsible for the `.card` block. According to the BEM methodology and separation of concerns, global styles should be placed in a separate file like `src/styles/utils/_global.scss` or `src/styles/index.scss`.